traffic_ops/app/db/migrations/2021010900000000_server_ip_profile_trigger_update.sql

```diff
@@ -0,0 +1,73 @@
+/*
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+-- +goose Up
+-- +goose StatementBegin
+CREATE OR REPLACE FUNCTION after_ip_address_table()
+    RETURNS TRIGGER
+AS
+$$
+DECLARE
+    server_count   BIGINT;
+    server_id      BIGINT;
+    server_profile BIGINT;
+    new_profile    BIGINT;
+BEGIN
+    SELECT profile INTO new_profile from server s WHERE s.id = NEW.server;
+    WITH server_ips AS (
+        SELECT s.id as sid, ip.interface, i.name, ip.address, s.profile, ip.server
+        FROM server s
+                 JOIN interface i
+                      on i.server = s.ID
+                 JOIN ip_address ip
+                      on ip.Server = s.ID and ip.interface = i.name
+        WHERE i.monitor = true AND ip.service_address = true
```

Comment:
if we limit it to only blocking when the ip address is a service address and the monitoring is true, then we probably need to add more triggers to check this again right? if the service address is changed then its covered by these since its in the ip_address table but we will need to add triggers for the interface table for if monitor is changed. might be worth a discussion if we want to allow duplicates if they are unmonitored (id say we probably want to block that case too)

Reply:
Talked with the team and turns out we dont need the `monitor = true` condition anymore.